### PR TITLE
IR-2532: Update Checkbox UI

### PR DESCRIPTION
**Pull request by CITIZENDOT**
## Summary

Replaces existing checkbox implementation with div & svg implementation for more control over styling.

## References
closes #_insert number here_

### CODE REVIEW

**Comment by aditya-mitra** [comment]
Will check this by tomorrow and test in on the studio

File: packages/ui/src/primitives/tailwind/Checkbox/index.tsx [hunk 1]
```diff
@@ -38,35 +40,23 @@ export interface CheckboxProps extends Omit<React.InputHTMLAttributes<HTMLInputE
...
   return (
-    <div className={twMerge('flex w-full items-center gap-4', containerClassName)}>
-      <input type="checkbox" onChange={(e) => onChange(e.target.value as any)} className={twClassName} {...rest} />
-      {label && (
-        <Label onClick={() => onChange(!value)} className="cursor-pointer self-stretch">
-          {label}
-        </Label>
+    <div
+      className={twMerge(
+        'grid h-[16px] w-[16px] cursor-pointer place-items-center rounded border border-theme-primary',
```
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Checkbox/index.tsx]: we should use tailwind relative sizes. in this case it should be `h-4 w-4`

File: packages/ui/src/primitives/tailwind/Checkbox/index.tsx [hunk 2]
```diff
@@ -38,35 +40,23 @@ export interface CheckboxProps extends Omit<React.InputHTMLAttributes<HTMLInputE
...
-      >
-        <polyline points="20 6 9 17 4 12" />
-      </svg>
+      onClick={() => {
+        if (!disabled) {
+          onChange(!value)
+        }
+      }}
+    >
+      {value && <HiCheck className="h-[12px] w-[12px] text-white" />}
```
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Checkbox/index.tsx]: same here - `h-3 w-3`

**Review by aditya-mitra — CHANGES_REQUESTED** [review]
Need to fix this

image

File: packages/ui/src/primitives/tailwind/Checkbox/index.tsx (hunk 1 above)
Comment (CITIZENDOT) [reply packages/ui/src/primitives/tailwind/Checkbox/index.tsx]: Done!

File: packages/ui/src/primitives/tailwind/Checkbox/index.tsx (hunk 2 above)
Comment (CITIZENDOT) [reply packages/ui/src/primitives/tailwind/Checkbox/index.tsx]: Done!

**Comment by CITIZENDOT** [comment]
fixed these issues

**Review by aditya-mitra — APPROVED** [review]
Awesome ✅

**Pull request merged**